Kickstarter-iOS/Views/Storyboards/Comments.storyboard

```diff
@@ -262,16 +262,12 @@
         <scene sceneID="BCA-NB-R56">
             <objects>
                 <viewController storyboardIdentifier="CommentDialogViewController" id="99a-92-DMl" customClass="CommentDialogViewController" customModule="Kickstarter_Framework" customModuleProvider="target" sceneMemberID="viewController">
-                    <layoutGuides>
```

Comment:
The `textView` and the `LoadingView` were previously constrained to the bottom layout guide. This caused an issue because the height of the keyboard always includes space that would technically be outside the "safe area".

Instead, pinned the bottom of these views to the superview's `bottom`, and manually adjusted the inset of the `bottom` constraint to account for the total height of the keyboard.